open-api/rest-catalog-open-api.yaml

```diff
@@ -3541,6 +3855,20 @@ components:
           schema:
             $ref: '#/components/schemas/LoadTableResult'
 
+    PrePlanTableResponse:
+      description: PrePlanTable Response
```

Comment:
nit: more consistent with other descriptions, like "result of preplanning a table"